api/sessions.py

```diff
@@ -0,0 +1,33 @@
+from sqlalchemy import MetaData, create_engine
+from flask import Flask
+from sqlalchemy.orm import sessionmaker
+
+if __package__ is None or __package__ == '':
+  from settings import DefaultSettings
+else:
+  from .settings import DefaultSettings
+
+app = Flask(__name__)
+app.config.from_object(DefaultSettings)
+
+user= str(app.config['POSTGRES_USER'])
+password= str(app.config['POSTGRES_PASSWORD'])
+host=str(app.config['POSTGRES_HOST'])
+port=str(app.config['POSTGRES_PORT'])
+db=str(app.config['POSTGRES_DB'])
+
+engine = create_engine('postgresql://' + user +':' + password +'@' + host +  ':' + port + '/' + db)
```

Comment:
This line looks redundant due to line 24

Reply:
Lol yeah, I'll remove that.